libflux/src/core/semantic/convert.rs

```diff
@@ -178,6 +176,17 @@ fn convert_member_assignment(stmt: ast::MemberAssgn, fresher: &mut Fresher) -> R
     })
 }
 
+fn convert_bad_statement(e: ast::BadStmt) -> Box<BadStmt> {
+    let loc = e.base.location;
+    let errors = vec![Error::ASTError(e.text, loc.clone())];
+    let statement = None;
```

Comment:
Should this match on `convert_statement` like the `convert_bad_expression` does?

Reply:
That doesn't yet exist yet on the ast statement. I just thought I would add the attribute ahead of time.

Reply:
Ok makes sense. I didn't realize the ast BadStatement didn't embed its own statement